tasks/system_probe.py

```diff
@@ -1511,6 +1516,22 @@ def cp_cmd(out_dir):
                 ctx.run(f"{sudo} find ./ -maxdepth 1 -type f -name '*.c' {cp_cmd('runtime')}")
 
 
+def copy_bundled_ebpf_files(
+    ctx,
+    arch: str | Arch = CURRENT_ARCH,
+):
+    # If we're bundling eBPF files, we need to copy the ebpf files to the right location,
+    # as we cannot use the go:embed directive with variables that depend on the build architecture
```

Comment:
Couldn't you have two different files that are architecture-specific that define the `bindata` variable?